File: app/src/main/java/com/android/mojtaba/androidbadgebutton/MainActivity.java
package com.android.mojtaba.androidbadgebutton;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import com.mojtaba_shafaei.androidToolbarBadgeButton.BadgeButton;

public class MainActivity extends AppCompatActivity {

@Override
protected void onCreate(Bundle savedInstanceState) {
super.onCreate(savedInstanceState);
setContentView(R.layout.activity_main);
final BadgeButton btn1 = findViewById(R.id.btnBadge);
final BadgeButton btn2 = findViewById(R.id.btnBadge2);
final BadgeButton btn3 = findViewById(R.id.btnBadge3);

btn1.setBadgeNum("9+");
btn1.setEnabled(false);

btn2.setBadgeNum("5");

btn3.setBadgeNum("16");
}
}

File: app/src/main/java/com/android/mojtaba/androidbadgebutton/MainActivity.kt
package com.android.mojtaba.androidbadgebutton

import android.graphics.Color
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.android.mojtaba.androidbadgebutton.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {
  private lateinit var binding: ActivityMainBinding

  override fun onCreate(savedInstanceState: Bundle?) {
    super.onCreate(savedInstanceState)
    binding = ActivityMainBinding.inflate(layoutInflater)
    setContentView(binding.root)

//    btn_badge1.setIcon(R.drawable.ic_phone_white_24dp)

    binding.btnBadge1.setBadgeNum("9+")
    binding.btnBadge1.isEnabled = false

    binding.btnBadge2.setBadgeNum("5")
    binding.btnBadge3.setBadgeNum("8")

    binding.btnBadge3.setOnClickListener{
      binding.btnBadge3.setBadgeTextColor(Color.RED)
      binding.btnBadge3.setBadgeBackgroundColor(Color.TRANSPARENT)
    }
  }
}
